accounts/keystore/keystore.go

```diff
@@ -51,6 +51,10 @@ var (
 	ErrLocked  = accounts.NewAuthNeededError("password or unlock")
 	ErrNoMatch = errors.New("no key for given address or file")
 	ErrDecrypt = errors.New("could not decrypt key with given password")
+
+	// ErrAccountAlreadyExists is returned if an account attempted to import is
+	// already present in the keystore.
+	ErrAccountAlreadyExists = errors.New("account alreaady exists")
```

Comment:
the typo guy missed the doble `a` alreaaaaaaady

Reply:
haha nice catch. do you mind if I fix this in the 1.9.16 branch so We don't re run the tests for this?

Reply:
Yes, of course. I just was hating the typos guy for the big PR and tried to point it out.